libbpfgo/helpers/kernel_features.go

```diff
@@ -0,0 +1,110 @@
+package helpers
+
+import (
+	"bufio"
+	"bytes"
+	"compress/gzip"
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"golang.org/x/sys/unix"
+)
+
+type KernelConfig map[string]string
+
+// NewKernelConfig simply creates the underlying map used by
+// the KernelConfig, you must use InitKernelConfig to read
+// in the values of the config
+func NewKernelConfig() KernelConfig {
```

Comment:
Why not merging this function with InitKernelConfig? Is there a meaning to call this function without calling InitKernelConfig after?

Reply:
I agree, meant to push this when I made the comment above.